### ISSUE TITLE
[flaky test][cmd/opampsupervisor] TestSupervisorStartsWithNoOpAMPServer failure

### ISSUE THREAD

**Issue by crobert-1** [open-telemetry/opentelemetry-collector-contrib#35033]
### Component(s)

cmd/opampsupervisor

### Describe the issue you're reporting

[Failing CI/CD action](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/10724047237/job/29739816002?pr=34889)

Failure output:
```
=== RUN   TestSupervisorStartsWithNoOpAMPServer
    e2e_test.go:158: 
        	Error Trace:	D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/cmd/opampsupervisor/e2e_test.go:158
        	            				D:/a/opentelemetry-collector-contrib/opentelemetry-collector-contrib/cmd/opampsupervisor/e2e_test.go:340
        	Error:      	Received unexpected error:
        	            	error loading config: yaml: line 15: did not find expected hexdecimal number
        	Test:       	TestSupervisorStartsWithNoOpAMPServer
    e2e_test.go:128: Shutting down
--- FAIL: TestSupervisorStartsWithNoOpAMPServer (5.02s)
```
Potentially related: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34704 was recently merged with changes to this area of code in the test.

**Comment by evan-bradley** [comment]
That's almost certainly it, thanks for opening this. I'll see if I can get a fix in quickly.

**Comment by crobert-1** [comment]
Resolved by #35035
